<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TTest10_2AClass</class>
 <widget class="QMainWindow" name="TTest10_2AClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TTest10_2A</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPlainTextEdit" name="txtEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>其他</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="TTest10_2A.qrc">
     <normaloff>:/TTest10_2A/Resources/new-file.png</normaloff>:/TTest10_2A/Resources/new-file.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="TTest10_2A.qrc">
     <normaloff>:/TTest10_2A/Resources/save.png</normaloff>:/TTest10_2A/Resources/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="TTest10_2A.qrc">
     <normaloff>:/TTest10_2A/Resources/help.png</normaloff>:/TTest10_2A/Resources/help.png</iconset>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="TTest10_2A.qrc">
     <normaloff>:/TTest10_2A/Resources/setting.png</normaloff>:/TTest10_2A/Resources/setting.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="TTest10_2A.qrc">
     <normaloff>:/TTest10_2A/Resources/open.png</normaloff>:/TTest10_2A/Resources/open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TTest10_2A.qrc"/>
 </resources>
 <connections/>
</ui>
